.changeset/eight-avocados-lick.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/react": patch
```

Comment:
This is technically a breaking change since we remove export but there is no usage internally at dotcom. Should we release potentially major? Or should we just do major and include in the major release?

Reply:
@broccolinisoup I think since tooltip is behind experimental/next it doesn't fall under semver so should be fine 👍 I would personally do `minor` since this is more around changing APIs than it would be fixing a bug but I don't know how important that is for these kind of "unstable" components 🤷

Reply:
You are totally right!